graylog2-web-interface/src/views/components/widgets/ExportWidgetAction/ExportWidgetPlug.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React from 'react';
+
+import { IconButton, OverlayTrigger } from 'components/common';
+
+const title = 'Export widget';
+const Explanation = () => (
+  <span>Export Aggregation widget feature is available for the enterprise version.
+    Graylog provides option to export your data into most popular file formats such as
+    CSV, JSON, YAML, XML etc.
```

Comment:
Shouldn't `Aggregation` be lowercase?
I think either `provides an option` or `provides options` sounds a bit better.

Reply:
What about:

```
Exporting widget results is an enterprise-only feature. With an enterprise license, you can export your results as Excel (xlsx), CSV, JSON, YAML or XML.

Please [click here] for more details.
```
With some helpful link included.